docs/examples/ingestion/document_management_pipeline.ipynb

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
<p>so it is a cache in memory or a data store in disk</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>remote caches and docstores are hosted somewhere else (like redis) and persist automatically </p><p> </p><p>So if you are using these, no need to save/load </p>